run_app.py

```diff
@@ -1,10 +1,13 @@
 # pylint: disable=cyclic-import
 """ run all of circles in one terminal """
 import logging
-from subprocess import Popen, check_call
-import threading
 import os
 import sys
+import threading
+from subprocess import Popen, check_call
+
+from dotenv import load_dotenv
```

Comment:
Where is this used?

Reply:
the threading and subprocess was already there; the `load_dotenv` is used to load up variables from `backend.env` as actual environment variables. very nice for not having to load up the file urself and write ur own parsing

Reply:
Nevermind oops